fem/bilininteg.cpp

```diff
@@ -19,6 +19,23 @@ using namespace std;
 
 namespace mfem
 {
+void BilinearFormIntegrator::Assemble(const FiniteElementSpace&)
```

Comment:
We can have come up with better names for these functions, for all new cases (EA PA MF)

Reply:
@vladotomov, do you feel that should be addressed in this PR, or could that be postponed?